### ISSUE TITLE
omnicored ( send-to-many ) docker image

### ISSUE THREAD

**Issue by wxf4150** [omnilaboratory/lnd#8]
run  send to many omnicored docker
```
docker run --name omnicored  -it  -p 18332:18332  -p 28332:28332  -p 28333:28333   -d --rm registry.cn-beijing.aliyuncs.com/omnicore/first:0.0.2
```
it export three ports, and port  28332 28333 is only used when our lnd  startup.
original lnd use btcd daemon not omnicored,  not need these ports  28332 28333 .

example: 
exec ./ob-lnd-debug \
    --noseedbackup \
    "--lnddir=~/apps/oblnd-dev" \
    "--$CHAIN.active" \
    "--$CHAIN.$NETWORK" \
    "--$CHAIN.node"="$BACKEND" \
    "--$BACKEND.rpchost"="43.138.107.248:**18332**" \
    "--$BACKEND.rpcuser"="$RPCUSER" \
    "--$BACKEND.rpcpass"="$RPCPASS" \
    "--rpclisten=$HOSTNAME:10009" \
    "--rpclisten=localhost:10009" \
    --listen=9735 \
    "--restlisten=0.0.0.0:8080" \
    --debuglevel="$DEBUG" \
    --bitcoind.zmqpubrawblock=tcp://43.138.107.248:**28332** \
    --bitcoind.zmqpubrawtx=tcp://43.138.107.248:**28333** \

# genarate host wallet 
```
> docker exec  omnicored  omnicore-cli -regtest getnewaddress
> 2N9DZN7HX532Wbm8YqLz28RrMbETjM2aXFL
```
2N9DZN7HX532Wbm8YqLz28RrMbETjM2aXFL is the host wallet address

# mine 
mine3 blocks to addree
```

docker exec  omnicored  omnicore-cli generatetoaddress 3  2N9DZN7HX532Wbm8YqLz28RrMbETjM2aXFL
```
#  faucet :
## send btc
```
omnicore-cli sendtoaddress $TargetAddress  1
```

# create asset
```
docker exec  omnicored omnicore-cli omni_sendissuancefixed "2N9DZN7HX532Wbm8YqLz28RrMbETjM2aXFL" 2 2 0 "t1" "" "ftoken" "baidu.com" "" 10000000000
```

## send asset
```
docker exec  omnicored   omnicore-cli "omni_send" 2N9DZN7HX532Wbm8YqLz28RrMbETjM2aXFL "$TargetAddress"  2147483653  100
```

**Comment by wxf4150** [comment]
i  should init the docker with an  init-asset . 

when  use the omnicored docker, it will  no need to create new asset manually

**Comment by neocarmack** [comment]
yes, agree.

**Comment by neocarmack** [comment]
I thought again. It is necessary letting user to issue new assets on an omnicored.  

Consider this senario: user tries to build multiple lightining network and trade tokens among them, he should not deploy many omnicored nodes as backends.

**Comment by wxf4150** [comment]
> docker exec  omnicored omnicore-cli omni_sendissuancefixed "2N9DZN7HX532Wbm8YqLz28RrMbETjM2aXFL" 2 2 0 "t1" "" "ftoken" "baidu.com" "" 10000000000

yes ,  user can issue new assets in the the docker now, .


i have add an init-asset which can startup a quick  test for developer.
the init-asset id is  2147483651.

**Fauct**  
and with below command can quick send user some test coin
```
docker exec omnicored  /bin/send_coin.sh mzP3GRXwkxF9GrfWBj755JP3DoitwjiRs6
# this command send user mzP3GRXwkxF9GrfWBj755JP3DoitwjiRs6  for one bitcoin , and 100 asset., and mine 3 blocks.
# one bitcoin is 100000000 satosh
```

```
cat /bin/send_coin.sh
omnicore-cli sendtoaddress $1  1
omnicore-cli omni_send mtowceAw2yeftR1pPg15QcsDqsnSik7Spz "$1"  2147483651  100
omnicore-cli generatetoaddress 3 mtowceAw2yeftR1pPg15QcsDqsnSik7Spz
omnicore-cli omni_getbalance  $1 2147483651
``` 

the docker image from version 0.0.3,will the init asset
```
docker run --name omnicored -it -p 18332:18332 -p 28332:28332 -p 28333:28333 -d --rm registry.cn-beijing.aliyuncs.com/omnicore/first:0.0.3
```

**Comment by wxf4150** [comment]
after you mine some block ,  use "docker stop  omnicored" and "docker start  omnicored"  command to restart the container , the new block data is persistent, and blocknum is right too.

but in the polar,  when you click "stop network" button, polar will use similar command like "docker rm omnicored"  to remove the container,  and the container data will be removed too.  when you  use polar start the network again ,the block data is reset.

it's a problom in polar.  i have pub a new version  0.0.5 fix the problom at commit 34659e269e27ca3e2a4a2a3059f9b186a0e73fc6
